### ISSUE TITLE
CI errors of gru_compute 

### ISSUE THREAD

**Issue by guoshengCS** [PaddlePaddle/Paddle#5563]
The following CI errors might be caused by the dependency lack of math_function in the CMake configurations of gru_compute, but I don't know why the errors didn't occur before. I am trying to fix it. sorry for this.

```
[23:47:35]HEAD is now at 649f04d... Added Pascal nvcc flags, bumped version
[23:47:49]In file included from /paddle/paddle/operators/math/gru_compute.cu:15:0:
[23:47:49]/paddle/paddle/operators/math/math_function.h:17:23: fatal error: mkl_cblas.h: No such file or directory
[23:47:49]compilation terminated.
[23:47:49]CMake Error at gru_compute_generated_gru_compute.cu.o.cmake:203 (message):
[23:47:49]  Error generating
[23:47:49]  /paddle/build/paddle/operators/math/CMakeFiles/gru_compute.dir//./gru_compute_generated_gru_compute.cu.o
[23:47:49]
[23:47:49]
[23:47:49]make[2]: *** [paddle/operators/math/CMakeFiles/gru_compute.dir/gru_compute_generated_gru_compute.cu.o] Error 1
[23:47:49]make[1]: *** [paddle/operators/math/CMakeFiles/gru_compute.dir/all] Error 2
[23:47:49]make[1]: *** Waiting for unfinished jobs....
[23:48:36]make: *** [all] Error 2
[23:48:36]Process exited with code 2
```

**Comment by luotao1** [comment]
You should merge the lateset code.

**Comment by guoshengCS** [comment]
Thank you ~   I have tried to fix it with #5564 , but the PR failed on CI for some other GPU reasons, and a similar PR #5567  is merged.
